redfish/config.py

```diff
@@ -43,14 +58,13 @@ def initialize_logger(REDFISH_LOGFILE,
     formatter = logging.Formatter(
         '%(asctime)s :: %(levelname)s :: %(message)s'
         )
+    f = open(os.path.expandvars(REDFISH_LOGFILE), 'w')
+    f.close()
+
```

Comment:
To my mind it is not needed. I guess you want to create the file, but that will be handled by RotatingFileHandler.
BTW, it is good practive to use the context manager "with" to open files. (I could explain you the benefits)